src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/TransitionBlock.cs

```diff
@@ -169,5 +172,19 @@ private sealed class LoongArch64TransitionBlock : TransitionBlock
             public override int OffsetOfFirstGCRefMapSlot => SizeOfCalleeSavedRegisters;
             public override int OffsetOfArgumentRegisters => OffsetOfFirstGCRefMapSlot;
         }
+
+        private sealed class Riscv64TransitionBlock : TransitionBlock
+        {
+            public static readonly TransitionBlock Instance = new Riscv64TransitionBlock();
+
+            public override int PointerSize =>  8;
+            public override int NumArgumentRegisters => 8;
+            public override int NumCalleeSavedRegisters => 12;
```

Comment:
Is `NumCalleeSavedRegisters` correct number? Please write the comments like other architectures.

Reply:
Fixed it